docs/source/linchpincli_linchpin_invgen.rst

```diff
@@ -1,24 +0,0 @@
-Invgen
```

Comment:
Cool. I'll at least get the stubs to match what I've done with the other commands and otherwise leave them blank.